### ISSUE TITLE
[XAML Controls Gallery(debug)->Basic Input->SplitButton]: Selected State of the color is not announced for color split button.

### ISSUE THREAD

**Issue by v-amnat** [microsoft/Xaml-Controls-Gallery#678]
**Tags**
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.2.1, ScreenReader,
 A11ySev2, XAMLControlsGallery, WinUI, Win32, Wipro

**Test Environment**
1. XAML Controls Gallery(debug)  version - 2.0.21.0
2. Win 32/ OS build- 21343.rs_Prerelease.210320-1757

**Steps to reproduce the bug**

1. Launch the XAML Controls Gallery (Debug) App
2. Go to the Basic Input-> SplitButton.
3. Hit on Tab key to traverse the "SplitButton(Carousals)".
4. Navigate to color split button and invoke it.
5. Listen the narrator announcement when navigated to currently selected color.

**Actual behavior**
With tab navigation on color split button, In collapsed mode Selected color is not announced by narrator. When split button is expanded using alt+down arrow key and navigated across the currently selected color, narrator is simply announcing as orange color instead of announcing orange color selected.

**Expected**
States, values, and properties of interactive components must be provided to assistive technology. Semantic HTML provides this information automatically. When using non-semantic components, this information can be provided by extending the markup with the appropriate ARIA, such as aria-expanded, aria-disabled, aria-current, etc. Learn more by reading about html/aria state and property mappings, as well as the states and adaptable properties list.

**User Impact** 
When the state, value or property of a component is not defined, assistive technology users are often blocked from using the control or knowing what to expect when interacting with the component.

**MAS Reference**
MAS 4.2.1 – Object Information

**Attachments**
[Color selected is not announced by narrator.zip](https://github.com/microsoft/Xaml-Controls-Gallery/files/6310122/Color.selected.is.not.announced.by.narrator.zip)

**Comment by chigy** [comment]
@anawishnoff fixed this
